ui-tests/tests/src/config.ts

```diff
@@ -41,5 +49,39 @@ export const config = {
       password: 'test',
     },
   },
-  voodooTimeoutMs: 30000,
+  voodooTimeoutMs: 30000
 };
+
+}else if(process.env.ENV === "other"){
```

Comment:
Just like we have an if above for "local" should we have an explicit if condition for PRODUCTION...just want to make sure that there is no confusion when we are dealing a a PRODUCTION environment.

Reply:
Hi @gibaros  these tests are not intented to be run in production env so I think we dont need to cover this point